src/PKSim.Core/Mappers/RangeChartDataToDataTableMapper.cs

```diff
@@ -42,16 +42,20 @@ protected override IEnumerable<DataRow> AddSpecificChartValues(DataRow row, Curv
          for (var i = 0; i < curveData.XValues.Count; i++)
          {
             var newRow = row.Table.NewRow();
+            var xValue = curveData.XValues[i];
```

Comment:
sometimes it's good to avoid a bit of code dup by introducing a variable lol...oh my